openedx/core/djangoapps/embargo/views.py

```diff
@@ -42,9 +43,12 @@ def get(self, request):
 
         if course_ids and user and user_ip_address:
             for course_id in course_ids:
-                if not check_course_access(CourseKey.from_string(course_id), user, user_ip_address):
-                    response['access'] = False
-                    break
+                try:
+                    if not check_course_access(CourseKey.from_string(course_id), user, user_ip_address):
+                        response['access'] = False
+                        break
+                except InvalidKeyError:
+                    return Response(data=None, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Return a message explaining the bad data here and below. Otherwise, developers will will have no clue what to do when they see these errors.